modules/ROOT/pages/install-a-bonita-bpm-cluster.adoc

```diff
@@ -70,21 +70,21 @@ IMPORTANT: If you don't use `bonita.platform.cluster.hazelcast.multicast.enabled
 For more information on this take a look at the  http://docs.hazelcast.org/docs/3.4/manual/html-single/hazelcast-documentation.html#hazelcast-cluster-discovery[Hazelcast Documentation].
 
 . Copy licenses of all your nodes in `setup/platform_conf/licenses`.
-. Run `setup.sh init` or `setup.bat init` as described in the  xref:bonita-bpm-platform-setup.adoc#init_platform_conf[platform setup tool page].
+. Run `setup.sh init` or `setup.bat init` as described in the  xref:bonita-platform-setup.adoc#init_platform_conf[platform setup tool page].
 
 At the end of the script, you should see the following line: "Initial configuration files successfully pushed to database".
 This creates the database tables needed by Bonita platform, stores the configuration into this database, and stores the license files for all your cluster nodes into the database.
 
-If later you need to change the configuration of the node discovery or add new licenses to the Bonita Platform configuration, you can update the configuration by following this  xref:bonita-bpm-platform-setup.adoc#update_platform_conf[guide].
+If later you need to change the configuration of the node discovery or add new licenses to the Bonita Platform configuration, you can update the configuration by following this  xref:bonita-platform-setup.adoc#update_platform_conf[guide].
```

Comment:
you can update the configuration by following this  xref:bonita-platform-setup.adoc#update_platform_conf[guide]. => you can update it by following this xref:bonita-platform-setup.adoc#update_platform_conf[guide].

Reply:
Done